### DISCUSSION TITLE
How to change featured image text color

### DISCUSSION THREAD

**Discussion by ZahraOsman** [CollectionBuilder/collectionbuilder.github.io#70] [Q&A] [answered]
Hi! Thank you again for everything you all do. 

Quick question: how do I change the default color on my featured image? Right now I have black text on the title "Friends of the Plum Blossom" against my image and I want it to be white so it shows up better. 

https://zahraosman.github.io/blossom/

**Comment by evanwill** [comment, marked as answer]
@ZahraOsman your Bootswatch theme `lux` is overriding the default white text color. To change it you can add your own custom CSS. 
Edit your ["_sass/_custom.scss"](https://github.com/ZahraOsman/blossom/blob/main/_sass/_custom.scss) file. In CSS, comments are between `/* and */`, so what is currently in your file is just a comment. Below the comment, add a new line with this CSS:

`.display-4 { color: #fff; }`

"display-4" is the class of the big text on the home page banner, `#fff` is the color white that matches your theme. If you want to try other colors, replace `#fff` with some other [color value](https://www.w3schools.com/colors/colors_picker.asp).
Have fun!

**Reply by ZahraOsman** [reply]
Evan! Thanks for casually saving my life and my final project for my class.
